packages/neos-ui-guest-frame/src/initializeGuestFrame.js

```diff
@@ -95,13 +95,20 @@ export default ({globalRegistry, store}) => function * initializeGuestFrame() {
             nodes
         });
 
-        window.requestIdleCallback(() => {
-            // only of guest frame document did not change in the meantime, we continue initializing the node
+        if ('requestIdleCallback' in window) {
```

Comment:
Might make sense to add some kind of polyfil for requestIdleCallback, as it's a good idea to use it in other places.
In the simplest form you could just wrap this check into a function, and put it somewhere into utility or smth.

Reply:
See https://developer.mozilla.org/en-US/docs/Web/API/Background_Tasks_API#Falling_back_to_setTimeout